server/grpc/server.go

```diff
@@ -92,7 +99,19 @@ func (s *Server) register() {
 	protobuf_api.RegisterOwnershipServer(s.instance, api.NewOwnershipServer(s.mc))
 	protobuf_api.RegisterRunnerServer(s.instance, api.NewRunnerServer(s.mc, s.b))
 
-	runner.RegisterRunnerServer(s.instance, runner.NewServer(s.mc.RPC, s.ep))
+	tokenToRunnerHash := &sync.Map{}
```

Comment:
would be great later to store this directly in the orchestrator so like that the orchestrator can know what are the running services fro early checks

Reply:
this can be injected in the New function of this file from the `core/main.go` and shared with orchestrator, if we need to.